README.md

```diff
@@ -172,9 +172,14 @@ We appreciate any responsible disclosure of vulnerabilities that might impact th
 
 ### Contributing
 
-The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome all contributions to the community and are excited to welcome you aboard.
+The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome all contributions to the community and are excited to welcome you aboard. Here is an overview of what you can contribute:
```

Comment:
`Here is an overview of what you can contribute:` should be `Here is an overview of what you can do to help:` This is clearer, cleaner, and flows better!

Reply:
As stated above this should be changed.

Reply:
What about `Here are some ways you can contribute:`